templates/build.gradle

```diff
@@ -111,7 +111,7 @@ configurations.all {
 }
 configurations {
     terasologyModule
-    compile.extendsFrom terasologyModule
+    implementation.extendsFrom terasologyModule
```

Comment:
Is the `terasologyModule` configuration doing anything at this point?

I don't see any other references to it. What would happen if we drop it entirely and put the module dependencies straight on `implementation` instead of adding them one place and then extending from it?

https://github.com/MovingBlocks/Terasology/blob/bd5670f9299766551fcd14859c938058371444b5/templates/build.gradle#L124

Reply:
We can. This for easy involve special configurations for modules (a bit overengineered)